service-discovery/README.adoc

```diff
@@ -31,304 +31,378 @@ WARNING: This content is generated by running npm --prefix .build run generate:a
+As a developer of applications and services, there might be cases where you need to connect applications deployed on a Kubernetes platform such as Red Hat OpenShift to Kafka instances created in {product-long}.
 
-OpenShift is an Enteprise Kuberentes Platform. When deploying Quarkus and any other type of application, developers can use
-the https://developers.redhat.com/blog/2019/12/19/introducing-the-service-binding-operator/[Service Binding Operator] to
-enable your applications to discover service configuration automatically in your OpenShift or Kubernetes instance.
+For example, suppose that you have two applications deployed to an OpenShift cluster:
 
-You can review all possible commands in https://github.com/redhat-developer/app-services-cli/blob/main/docs/commands/rhoas_cluster.adoc[cli documentation].
+* One application that publishes price updates for a variety of stocks
+* A second application that consumes the price updates for publication on a web page
 
+In addition, you have a Kafka instance in {product}. Each time there is a price update, you want to use the Kafka instance to forward the update as an event to the consuming application.
 
-In this Quick Start, you'll connect your OpenShift Streams for Apache Kafka cloud service to your OpenShift cluster using the Red Hat OpenShift Application Services. (RHOAS) Operator and CLI. For more information on Red Hat OpenShift Application Services, please visit link:https://cloud.redhat.com/application-services[cloud.redhat.com.]
+To achieve this behavior, you need a way to connect the applications in OpenShift to your Kafka instance in {product}. Connecting a Kubernetes application to a backing service such as {product} is called __service binding__.
 
-.Prerequisites
-ifndef::community[]
-* You have a Red Hat account.
-endif::[]
-* You have a running Kafka instance in {product}.
+This guide describes how to use the OpenShift Application Services (RHOAS) Operator to perform service binding for {product}. The Kubernetes platform used throughout this guide is Red Hat OpenShift.
 
-// Condition out QS-only content so that it doesn't appear in docs.
-// All QS anchor IDs must be in this alternate anchor ID format `[#anchor-id]` because the ascii splitter relies on the other format `[id="anchor-id"]` to generate module files.
-ifdef::qs[]
-[#description]
-Using the {product-long} services in your OpenShift cluster{product-long}.
+[id="con-about-service-binding-using-rhoas-operator_{context}"]
```

Comment:
@jbyrne-redhat To fix the mod doc build issue, add one more line break before this anchor ID. It's a silly/mysterious problem with the splitter dealing with the first module of the set. I've reported it here: https://github.com/LightGuard/asciidoc-splitter/issues/89

Reply:
@sterobin Thanks for recognizing and providing some info about this bug. The suggested fix worked nicely!